packages/node_modules/@node-red/editor-client/src/js/ui/contextMenu.js

```diff
@@ -152,8 +152,16 @@ RED.contextMenu = (function() {
             }
 
         }
+
+        var direction = "right";
+        var MENU_WIDTH = 600; // can not use menu width here
+        if ((options.x -$(document).scrollLeft()) >
+            ($(window).height() -MENU_WIDTH)) {
```

Comment:
```suggestion
            ($(window).width() -MENU_WIDTH)) {
```

should be `width` right?

Reply:
Sorry.  Was stupid mistake.